mozjs-sys/src/jsgc.rs

```diff
@@ -153,6 +169,14 @@ pub trait GCMethods {
     unsafe fn post_barrier(v: *mut Self, prev: Self, next: Self);
 }
 
+impl<T> GCMethods for Option<T> {
+    unsafe fn initial() -> Self {
+        None
+    }
+
+    unsafe fn post_barrier(_v: *mut Self, _prev: Self, _next: Self) {}
+}
+
```

Comment:
Shouldn't we use bounds here (`T: GCMethods`) and call underlying methods (`T::post_barrier`)?